src/_scss/pages/reviewData/_content.scss

```diff
@@ -8,4 +8,9 @@
     @import "./rightSide/rightSide";
     @import "./bottomSide/bottomSide";
     @import "./narrative/narrative";
+
+    .monthly-submission-error {
```

Comment:
Please check margin. It has an extra left spacing on smaller screen. Might want to move the error to the same container with certify and notify button`div class="submission-wrapper"`?